admin/app/views/workarea/admin/pricing_skus/_cards.html.haml

```diff
@@ -14,6 +14,7 @@
             %li
               %strong #{t('workarea.admin.fields.tax_code')}:
               = model.tax_code.presence || '(none)'
+            = append_partials('admin.card_cost_and_margin', model: model) 
```

Comment:
Looks like we already have an append point for this list at `admin.pricing_sku_attributes_card`. I think in base, we'd consider that sufficient coverage here.

Reply:
The card has a similar layout issue that I have with the edit and new form (posted below). They don't all exactly match in order. The append point is also out of order with the other layouts. This picture it utilizing the current append point in base, which puts the new fields at the bottom and that doesn't make sense to me for those to be in different locations on all three places.

<img width="341" alt="Screen Shot 2020-02-06 at 8 08 52 PM" src="https://user-images.githubusercontent.com/37818357/73994856-94a56280-491c-11ea-9e68-ce96d71cdc8d.png">